po/wesnoth-anl/bg.po

```diff
@@ -96,14 +96,14 @@ msgstr "Мал Шики"
 #. [side]
 #: data/multiplayer/scenarios/4p_A_New_Land.cfg:246
 msgid "teamname^Prisoners"
-msgstr "teamname^Затворници"
+msgstr "Затворници"
 
 #. [side]
 #: data/multiplayer/scenarios/4p_A_New_Land.cfg:270
 #, fuzzy
 #| msgid "teamname^Prisoners"
 msgid "teamname^Monsters"
-msgstr "teamname^Затворници"
+msgstr "Затворници"
```

Comment:
Same translation as "Prisoners", probably incorrect.

Reply:
Another fuzzy translation that needs to be updated - previously was 'Prisoners'. Leaving as-is.